datumaro/plugins/yolo_format/converter.py

```diff
@@ -399,3 +401,88 @@ def _make_annotation_line(self, width: int, height: int, skeleton: Annotation) -
             points_values[position] = f"{x:.6f} {y:.6f} {element.visibility[0].value}"
 
         return f"{self._map_labels_for_save[skeleton.label]} {bbox_string_values} {' '.join(points_values)}\n"
+
+
+class YOLOv8ClassificationConverter(Converter):
+    # https://github.com/AlexeyAB/darknet#how-to-train-to-detect-your-custom-objects
```

Comment:
The url seems to be unrelated to YOLOv8. I think it would make sense to add a similar one for v8.

Reply:
Removed the url.
There is already a link to official docs for yolov8 in datumaro docs, so i dont see a point of adding a link to some unofficial guide here (also I did not manage to find something like this)